tests/benchmark/codes/README.md

```diff
@@ -0,0 +1,15 @@
+# Guide to benchmark codes
+
```

Comment:
Are there are benchmarks that we *weren't* able to include code for? If so, these should be listed.

Reply:
Not sure, will defer this to @elisachisari

Reply:
Done